networkx/algorithms/bipartite/extendability.py

```diff
@@ -0,0 +1,133 @@
+""" Provides a function for computing the extendability of a graph which is
+undirected, simple, connected and bipartite and contains at least one perfect matching."""
+
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+
+
+__all__ = ["k_extendability"]
+
+
+@not_implemented_for("directed")
+@not_implemented_for("multigraph")
+def k_extendability(G):
+    """Computes the extendability of a graph.
+
+    Definition:
+    Graph G is $k$-extendable if and only if G has a perfect matching and every
+    set of $k$ independent edges can be extended to a perfect matching in G.
+
+    Definition:
+    The extendability of a graph, denoted by ext(G), is defined as the maximum
+    $k$ for which G is $k$-extendable.
```

Comment:
```suggestion
    The extendability of a graph is defined as the maximum $k$ for which `G`
    is $k$-extendable. Graph `G` is $k$-extendable if and only if `G` has a
    perfect matching and every set of $k$ independent edges can be extended
    to a perfect matching in `G`.
```
Just a suggestion to reorganize this information to make it a bit more digestible. This summary part of the docstring should really focus on the information *in the context of the networkx function*.